lldbinit.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+# Initialization for lldb.
+
+__copyright__ = """
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""
+
+if __name__ == "__main__":
+    print("Run only as script from LLDB... Not as standalone program!")
+
+import lldb  # type: ignore
+import os
+
+project_root = os.path.dirname(os.path.realpath(__file__))
+
```

Comment:
Unused?
```suggestion
```

Reply:
Oh, I meant to use this instead of os.getcwd()